.mergify.yml

```diff
@@ -0,0 +1,18 @@
+---
+pull_request_rules:
+  - actions:
+      merge:
+        method: rebase
+        rebase_fallback: null
+        strict: true
+    conditions:
+      - label=ready-to-merge
+      - '#approved-reviews-by>=1'
+      - status-success=continuous-integration/travis-ci
+      - status-success=docs/readthedocs.org:crate-pdo
+    name: default
+  - name: Delete branch after merge
```

Comment:
shall we use the github option instead? we experienced a bit faster responses and reliability on the crate repository.

Reply:
oh could do. I was copying this from elsewhere. I will remove

Reply:
thanks!